src/witnet/request.js

```diff
@@ -15,6 +15,7 @@ class Request {
       commit_fee: 0,
       reveal_fee: 0,
       tally_fee: 0,
+      collateral: null,
```

Comment:
As per WIP0002, this should default to the minimum collateral (`1_000_000_000`).